sample/library_manager_v2/library_manager.py

```
@@ -59,6 +59,9 @@ def library_manager():
 
         elif response == "p":
 
+            if (len(basket) > 5):
+                raise Exception("Basket is too heavy, I can't bring it to the shelf.")
```

Ruff (0.14.8)

Create your own exception

(TRY002)